core/documentation/introduction/Welcome.mdx

```diff
@@ -1,8 +1,21 @@
 import { Meta } from "@storybook/blocks";
+import json from "../../../package.json";
 
-<Meta title="Documentation/Introduction/Introduction" />
+<Meta title="Documentation/Introduction/Welcome" />
 
-# Introduction
+<div class="full-width-bg">
+# Core Style Library
+
+</div>
+
+<div class="version-badges">
+  <span class="fudis-badge fudis-badge__primary">
+    [NPM v{json.version}](https://www.npmjs.com/package/@funidata/fudis-core)
```

Comment:
I would rename this from `json` to `pkg` which is the NPM way of referencing a package.